logical_switch_lb.go

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) 2017 eBay Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package goovn
+
+import (
+	"fmt"
+	"github.com/ebay/libovsdb"
+)
+
+
+func (odbi *ovnDBImp) lslbAddImp(lswitch string, lb string) (*OvnCommand, error) {
+	var operations []libovsdb.Operation
+	namedUUID, err := newRowUUID()
+	if err != nil {
+		return nil, err
+	}
+	mutateUUID := []libovsdb.UUID{{namedUUID}}
```

Comment:
need check provided uuids i think

Reply:
ok like check if uuid exists?

Reply:
yes, i think ok like check for both uuids is fine

Reply:
Done